### PR TITLE
Tutorialtemplate

### LINKED ISSUE #2: Tutorial Template
## Value statement - a requirement, not a solution

As a user  **of the MindReset App**
I need **a instructions**
so that **I can to bring my thoughts to rest.**

Bildschirmfoto 2020-11-16 um 12 15 12

## Acceptance criteria
- [x] Image, Titel and Description always visible
- [x] Image, Titel and Description placed in center

## Tasks
- [x] Create Feature Branch
- [x] Create Tutorial Template Component in Story Book
- [x] Add Component to App
- [x] Check styling at devices (ios, android, versel)
- [x] Add missing styling
- [x] Testing devices (ios, android, Versel)
 **Optional** Bug fixing from testing devices (ios, android, versel)

## Size
medium

### PR DESCRIPTION
**Pull request by dajanaheinze**

### CODE REVIEW

**Review by mmurakaru — APPROVED** [review]
Looks good!

File: public/index.html [hunk 1]
```diff
@@ -25,6 +25,9 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
```
Comment (marwinburesch) [inline public/index.html:27]: You should change the title to something else, but in another PR probably

File: public/index.html [hunk 2]
```diff
@@ -25,6 +25,9 @@
...
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Work+Sans:ital,wght@0,100;0,200;0,300;0,400;0,600;1,100;1,200;1,300&display=swap');
+      </style>
```
Comment (marwinburesch) [inline public/index.html:30]: If you want to use the `@import` syntax please add fonts via `<GlobalStyles>`, otherwise use the `<link>` element here instead.
Example: 
image

File: src/App.js [hunk 3]
```diff
@@ -1,11 +1,23 @@
 import styled from 'styled-components/macro';
+import TutorialTemplate from './TutorialTemplate'
+
+
```
Comment (marwinburesch) [inline src/App.js:4]: Unnecessary whitespace

File: src/GlobalStyles.js [hunk 4]
```diff
@@ -1,17 +1,29 @@
 import { createGlobalStyle } from 'styled-components/macro';
-@import url('https://fonts.googleapis.com/css2?family=Work+Sans:ital,wght@0,100;0,200;0,300;0,400;0,600;1,100;1,200;1,300&display=swap');
 
 export default createGlobalStyle`
-  *{
+*, ::after, ::before {
```
Comment (marwinburesch) [inline src/GlobalStyles.js:4]: nice!

File: src/GlobalStyles.js [hunk 5]
```diff
@@ -1,17 +1,29 @@
...
+    --primary-text: #080808;
+    --primary-border: #F2F2F2;
+    --primary-shadow: #8c8d80;
+    --secendary-shadow: #bebfae;
+
+    --primary-black: #0D0D0D;
+    --primary-anthrazit: #262626;
+    --primary-white:  #F2F2F2;
+
+}
```
Comment (marwinburesch) [inline src/GlobalStyles.js:28]: excellent idea 👍

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Good work :) I am nitpicking 
🐜

File: src/GlobalStyles.js [hunk 6]
```diff
@@ -1,17 +1,29 @@
...
- 
-  body {
-    margin: 0;
-  }
-`;
\ No newline at end of file
+body {
+    margin:0;
+    font-size: 112.5%;
+    background-color: var(--primary-background);
```
Comment (badabam) [inline src/GlobalStyles.js:12]: Nice! I like that you already use css variables.

File: src/GlobalStyles.js (hunk 4 above)
Comment (badabam) [reply src/GlobalStyles.js:4]: That's better than what I usually do. Nice!

File: src/TutorialTemplate.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+import styled from 'styled-components/macro';
+import meditation from './assets/image-meditation.jpg'
+
+
+export default function TutorialTemplate (){
+    return (
+    <TutorialWrapper>
+        <img src={meditation} alt=""/>
+        <h2>5 Minuten of...</h2>
+        <p>Lorem ipsum dolor sit amet consectetur Quos accusamus libero minus ullam, beataelibero minus ullam, beatae possimus nostr</p>
```
Comment (badabam) [inline src/TutorialTemplate.js:11]: Please try to avoid placeholder text. Write a first version of some real text that you can iterate on later.

File: src/TutorialTemplate.js [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+    </TutorialWrapper>
+    )   
+}
+
+const TutorialWrapper = styled.div`
+    display: grid;
+    border: 1px solid var(--primary-border);
+    background-color: var(--secendary-background); 
+    box-shadow:  20px 20px 60px var(--primary-shadow),
+                -10px -10px 50px var(--secendary-shadow);
```
Comment (badabam) [inline src/TutorialTemplate.js:21]: Double shadow, nice. But it's "secondary", not "secendary"

File: src/TutorialTemplate.js [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+        text-align: center;
+        text-transform: uppercase;
+    }
+
+    p {
+        font-size: 0.8em;
+        font-weight: 200;
+        margin: 0;
+        padding: 0 10px 0 10px;
+        text-align: center;
```
Comment (badabam) [inline src/TutorialTemplate.js:46]: I like your sorting of the css declarations. 👍

File: src/GlobalStyles.js [hunk 10]
```diff
@@ -1,17 +1,29 @@
...
+    color: var(--primary-text);
+}
+
+ :root {
```
Comment (badabam) [inline src/GlobalStyles.js:16]: Please consider defining your colors in the `:root` above the `body`. Or even make it an extra global style component: `<Colors />` or `<CSSVariables />`

**Review by badabam — COMMENTED** [review]
What a nice first PullRequest. I have some minor remarks, please have a look.

We talked about you leaving out styleguidist/storybook and tests for now – so this is fine. 

Keep on the good work! 🚀
